### PR TITLE
Things

### PR DESCRIPTION
**Pull request by IRacle1**
Some little things i wanted to pr before dildo

 - `ItemType::AntiScp207` as `Consumable` in `Item::Create` 🥱
 - `Throwable::Scale` working with throwing item🥰
 - `ServerOnUsingCompleted` event in some usable items things, probably fixing something
 - ~`is not`~ `!=` 👍

### CODE REVIEW

**Review by Jesus-QC — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

File: EXILED/Exiled.API/Features/Items/Throwable.cs [hunk 1]
```diff
@@ -45,6 +45,21 @@ internal Throwable(ItemType type, Player player = null)
...
+                return base.Scale;
+            }
+
+            set
+            {
+                base.Scale = value;
+                Projectile.Scale = value;
+            }
+        }
+
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Items/Throwable.cs:62]: can you inline this code ?

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.API/Features/Items/Throwable.cs [hunk 2]
```diff
@@ -45,6 +45,21 @@ internal Throwable(ItemType type, Player player = null)
         {
         }
 
+        /// <inheritdoc/>
+        public override Vector3 Scale
+        {
+            get
+            {
+                return base.Scale;
+            }
```
Comment (Jesus-QC) [inline EXILED/Exiled.API/Features/Items/Throwable.cs:54]:
```suggestion
            get => base.Scale;
```

File: EXILED/Exiled.API/Features/Items/Throwable.cs [hunk 3]
```diff
@@ -45,6 +45,21 @@ internal Throwable(ItemType type, Player player = null)
...
+
```
Comment (Jesus-QC) [inline EXILED/Exiled.API/Features/Items/Throwable.cs:55]:
```suggestion
```

**Review by Jesus-QC — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.API/Features/Items/Throwable.cs (hunk 1 above)
Comment (IRacle1) [reply EXILED/Exiled.API/Features/Items/Throwable.cs:62]: yeah

**Review by Jesus-QC — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Review by louis1706 — APPROVED** [review]

**Pull request merged**